src/Silk.Data/MediatR/TagRequest.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Collections.Generic;
+using MediatR;
+using Silk.Data.Models;
+
+namespace Silk.Data.MediatR
+{
+    public class TagRequest
+    {
+        public record  Get(string Name, ulong GuildId) : IRequest<Tag?>;
```

Comment:
Nit, excess space here.  Get some kind of "format-on-save" extension to save yourself headaches.

Reply:
Alas the one of few issues with Rider. I can format tomorrow when I'm back on my machine.